README.md

```diff
@@ -6,21 +6,20 @@ Homepage: https://godotengine.org
 
 #### 2D and 3D cross-platform game engine
 
-Godot Engine is a feature-packed, cross-platform game engine to create 2D and
-3D games from a unified interface. It provides a comprehensive set of common
-tools, so that users can focus on making games without having to reinvent the
-wheel. Games can be exported in one click to a number of platforms, including
-the major desktop platforms (Linux, Mac OSX, Windows) as well as mobile
-(Android, iOS) and web-based (HTML5) platforms.
+Godot Engine is a feature-packed, cross-platform game engine used to create 2D
+and 3D games from a unified interface. Godot provides a comprehensive set of
+common tools that allow users to focus on making games without having to
+reinvent the wheel. Games can be exported in one click to a number of platforms,
+including the major desktop platforms (Linux, Mac OSX, Windows) as well as 
+mobile (Android, iOS) and web-based (HTML5) platforms.
 
 #### Free, open source and community-driven
 
 Godot is completely free and open source under the very permissive MIT license.
-No strings attached, no royalties, nothing. The users' games are theirs, down
+No strings attached, no royalties, nothing. Godot users' games are theirs, down
 to the last line of engine code. Godot's development is fully independent and
-community-driven, empowering users to help shape their engine to match their
-expectations. It is supported by the Software Freedom Conservancy
-not-for-profit.
+community-driven, empowering users to shape the engine to their expectations.
+Godot is supported by the Software Freedom Conservancy not-for-profit charity.
```

Comment:
I would keep "It" here too. Else you have 4 times Godot at the beginning of a sentence in one paragraph, which is quite a heavy style

Reply:
That's a good point, thanks.